modules/ROOT/pages/permissions-by-product.adoc

```diff
@@ -1,9 +1,6 @@
 = Permissions Available in Anypoint Platform
-ifndef::env-site,env-github[]
-include::_attributes.adoc[]
-endif::[]
 
-Anypoint Platform has a variety of permissions that control user access to various areas of the software. While each product owns its own permissions, most of the following permissions can be assigned to xref:teams.adoc[teams], granted to xref:users.adoc#grant-user-permissions[individual users], or combined to create xref:roles.adoc#create-custom-roles[roles (deprecated)]. Others can be assigned in their respective product interfaces. Additionally, some permissions are configurable at the environment or business group level. 
+Anypoint Platform has a variety of permissions that control user access to various areas of Anypoint Platform. Each product owns its own permissions, but you can assign most of the following permissions to xref:teams.adoc[teams] and xref:users.adoc#grant-user-permissions[individual users]. You can assign other permissions in their respective product interfaces. 
 
 Some products require permissions from other products to use them properly. For example, Anypoint Monitoring requires users to have certain Runtime Manager permissions in addition to Anypoint Monitoring-related permissions. See each product's documentation to determine which permissions your users need and how to set them.
```

Comment:
```suggestion
Some products require permissions from other products to use them properly. For example, Anypoint Monitoring requires users to have certain Runtime Manager permissions in addition to Anypoint Monitoring-related permissions. See each product's documentation to determine which permissions your users need and how to set them.
```

Not anything you added, but I don't understand what "to use them properly" means. "for the permission to work" or something?